css-basics_pokemon-page/css/styles.css

```diff
@@ -0,0 +1,116 @@
+/* The exercise will start here! Sometimes you may need to add a class attribute to some elements in your HTML structure, but otherwise it won't be necessary to adjust the HTML to solve the tasks. */
+
+/* ------- General Styles ------- */
+
+/* Task 1: First of all we want to adjust how the size of our elements is calculated when working with CSS. Do you know how to switch from "content-box" to "border-box"? */
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+/* Task 2: Let's work on some basic stylings that we want to apply to all elements of your code via inheritance. Which selector do we need for this? */
+
+/* add selector */
+* {
```

Comment:
I'm not done with it yet but I wouldn't use the selector * twice in the code. Maybe body?

Reply:
Thanks Victor!!